### PR TITLE
Alpha vector policies work with arbitrary beliefs

### PR DESCRIPTION
**Pull request by zsunberg**

### CODE REVIEW

**Comment by zsunberg** [comment]
@MaximeBouton I changed the implementation to have less repeated code and work for more beliefs. It now creates a sparse vector from a SparseCat to dot with the alpha vectors instead of using the for loop. I don't think this should have affected performance very much. Can you think of anything it would effect?

File: src/alpha_vector.jl [hunk 1]
```diff
@@ -76,45 +76,26 @@ function actionvalues(p::AlphaVectorPolicy, b::DiscreteBelief)
     return max_values
 end
 
-function value(p::AlphaVectorPolicy, b::SparseCat)
-    maximum(sparsecat_dot(p.pomdp, a, b) for a in p.alphas)
-end
+"""
```
Comment (MaximeBouton) [inline src/alpha_vector.jl:79]: I think it is great thanks! 
Should `beliefvec` belong to BeliefUpdaters.jl?

Comment (MaximeBouton) [reply src/alpha_vector.jl:79]: Could you add the docstring to the documentation? Thanks

**Comment by zsunberg** [comment]
@MaximeBouton can you take a look at this real quick so I can merge it and tag a version? I want to be sure I'm not messing anything up.

**Comment by MaximeBouton** [comment]
Sorry I forgot to submit my review yesterday!

File: src/alpha_vector.jl (hunk 1 above)
Comment (zsunberg) [reply src/alpha_vector.jl:79]: Hmm... yeah, not sure where it should be. I'll merge this now, but see also JuliaPOMDP/POMDPs.jl#233

**Pull request merged**